### PR TITLE
Story list fragment ready to merge with Master

### PR DESCRIPTION
**Pull request by paragsagar**
Please review 

@ns0714
@yangyzheng
@fcduarte

### CODE REVIEW

**Comment by ns0714** [comment]
Looks good. Just some formatting issues in StoryArrayAdapter. Can you merge to master

File: project.properties [hunk 1]
```diff
@@ -12,4 +12,5 @@
 
 # Project target.
 target=android-19
-android.library.reference.1=../../adt-bundle-windows-x86-20140702/sdk/extras/android/support/v7/appcompat
+android.library.reference.1=../../../android/sdk/extras/android/support/v7/appcompat
```
Comment (fcduarte) [inline project.properties:15]: A question: are you guys using any resources from appcompat? If not you can just copy the jar file to `libs` folder. If you are using please follow this guide:

https://developer.android.com/tools/support-library/setup.html

The way it is now is forcing all developers to have the same folder structure for the SDK and the project. The guide shows how to add a project to your current one and copy appcompat's source folder.

File: res/values/app_config.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (fcduarte) [inline res/values/app_config.xml:1]: Same problem that @yangyzheng was having .. Please remove this file from your branch.

File: src/com/tresleches/aadp/adapter/StoryArrayAdapter.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
+package com.tresleches.aadp.adapter;
```
Comment (fcduarte) [inline src/com/tresleches/aadp/adapter/StoryArrayAdapter.java:1]: File has some formatting issues, just run a cmd + shift + f on Eclipse.

File: src/com/tresleches/aadp/adapter/StoryArrayAdapter.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+		super(context, resource, objects);
+		
+	}
+	@Override
+	public View getView(int position, View view, ViewGroup parent) {
+		final Story story = getItem(position);
+		
+		
+		if(view == null)
+		{
```
Comment (fcduarte) [inline src/com/tresleches/aadp/adapter/StoryArrayAdapter.java:31]: Hmm, this is more a C coding style. Java follows this convention:

```
if (something) {
    doSomething();
}  else {
    doOtherStuff();
}
```

File: src/com/tresleches/aadp/model/Story.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+@ParseClassName("Story")
+public class Story extends ParseObject {
+
+	public String getName() {
+		return getString("name");
```
Comment (fcduarte) [inline src/com/tresleches/aadp/model/Story.java:14]: Same comment that I gave to @yangyzheng, the field string could be extract to a public constant.

File: src/com/tresleches/aadp/fragment/StoryFragment.java [hunk 6]
```diff
@@ -1,9 +1,86 @@
 package com.tresleches.aadp.fragment;
 
+import java.util.ArrayList;
```
Comment (fcduarte) [inline src/com/tresleches/aadp/fragment/StoryFragment.java:3]: Formatting issues here too.

**Pull request merged**